certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          },
+          {
+            "x-ms-summary": "Note",
+            "name": "note",
+            "in": "body",
+            "required": true,
+            "description": "Note content",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "successfully created"
+          }
+        }
+      }
+    },
+    "/orders/{orderId}/documents": {
+      "post": {
+        "tags": [
+          "Document"
+        ],
+        "summary": "Add Document",
+        "description": "Add a document to an existing RamQuest order",
+        "operationId": "AddDocumentToOrder",
+        "consumes": [
+          "text/plain"
+        ],
+        "produces": [
+          "text/plain"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "orderId",
+            "x-ms-summary": "File #",
+            "description": "Specify the File # to which to add the document",
+            "required": true,
+            "type": "string",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "in": "query",
+            "name": "fileName",
+            "x-ms-summary": "Document Name",
+            "description": "Specify a name for the document including its file extension e.g. document_name.pdf",
+            "type": "string",
+            "required": true
+          },
+          {
+            "in": "body",
+            "name": "fileContent",
+            "x-ms-summary": "Document Content",
+            "description": "Document content from cloud storage connector (Microsoft OneDrive, Dropbox, etc.)",
+            "schema": {
+              "type": "string",
+              "format": "binary"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PubCS.TitleOrder.Api.Models.DateType": {     
+      "description": "Gets or Sets DateType",
+      "type": "string"
+    },
+    "PubCS.TitleOrder.Api.Models.Date": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.DateType"
+        },
+        "value": {
+          "format": "date-time",
+          "description": "Gets or Sets Value",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.PartyReference": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "partyIdReference": {
+          "description": "Gets or Sets PartyIdReference",
+          "type": "integer"
+        },
+        "nickName": {
+          "description": "Gets or Sets NickName",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderDetail": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.